doc/getting_started.md

```diff
@@ -0,0 +1,25 @@
+# Getting started with PacSim
+
+To use PacSim, you need to create a message converter node to match your own interfaces with the simulator. You will need to advertise and publish to some topics. Note that some topics are not directly used in the simulator.
+
+| TOPIC                        | MESSAGE TYPE               | DESCRIPTION                                                                           | USED?  |
+|------------------------------|----------------------------|---------------------------------------------------------------------------------------|---|
+| /pacsim/steering_setpoint    | pacsim::msg::StampedScalar | Target steering angle at the steering wheel/sensor (rad)                              | YES |
+| /pacsim/torques_max          | pacsim::msg::Wheels        | Upper bound torque value at each wheel's motor (Nm)                                   | YES |
+| /pacsim/powerground_setpoint | pacsim::msg::StampedScalar | Powered ground multiplier. 0 for no powered aero, 1 for full capacity (Dimensionless) | YES |
```

Comment:
Please put this in the last row so that related topics are grouped together

Reply:
Sorry, I don't fully understand the order in which you want me to put these topics, but I will recommit with a newly sorted one and see if you like it. If you don't like it, let me know.